### PR TITLE
Adding new functionality: vlan, vlanrange, vlanview, next available vlan,  forward zone, delegated zone and network container

### PR DESCRIPTION
**Pull request by IngmarVG-IB**
Support added for following wapi objects: 

1. vlan, vlanrange, vlanview, next available vlan
2. forward zone, delegated zone (
- including reverse zones ipv4 and ipv6)
3. network containers
- ipv4 and ipv6

### CODE REVIEW

**Comment by saiprasannasastry** [comment]
I am on time off today will look into it once I am back to the office. Apologies for the delay

File: infoblox_client/objects.py [hunk 1]
```diff
@@ -1050,3 +1052,131 @@ class CNAMERecord(InfobloxObject):
...
+
+    def __repr__(self):
+        return "VlanAllocation: {0}".format(self.next_available_vlan_id)
+
+    def __str__(self):
+        return str(self.next_available_vlan_id)
+
+    @classmethod
+    def next_available_ip_from_vlanrange(cls, vlanview, vlanrange):
+        return cls(vlanrange, 'func:nextavailablevlanid:'
```
Comment (JonasKs) [inline infoblox_client/objects.py]: Visual indent PEP8:

```py
        return cls(vlanrange, 'func:nextavailablevlanid:'
                              '{vlanrange:s},'
                              '{vlan_view_name:s}'.format(**locals()))
````

**Comment by yuewko** [comment]
@JonasKs 

> Travis still fails on Python 3.4, but that's unrelated to this PR.

I believe I have a fix for that: https://github.com/infobloxopen/infoblox-client/pull/225
I'll rerun CI to verify once it's merged.

**Comment by yuewko** [comment]
@IngmarVG-IB ,
> Travis still fails on Python 3.4, but that's unrelated to this PR

I "think" you have to rebase this PR to include the changes from #225 in order to fix the above mentioned Travis failure. (this is in addition to fixing the PEP8 stuff)

Please give it a try when you get a chance. Thanks!

**Comment by IngmarVG-IB** [comment]
added the jarvis fix and bumped it to latest version

File: setup.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 setup(
     name='infoblox-client',
-    version='0.4.22',
+    version='0.4.23',
```
Comment (JonasKs) [inline setup.py]: Feel like this would be `0.5.0`?

Comment (saiprasannasastry) [reply setup.py]: yes, even i feel this should be 0.5.0 as it is a new feature addition

**Comment by JonasKs** [comment]
Any followup here? The PyPi is very outdated compared to `master`, and this seems like a good PR to bump the versioning. 

@saiprasannasastry @yuewko

**Comment by saiprasannasastry** [comment]
@JonasKs PM teams are talking on this, and they have told me to hold tihs. havent got an update. hence not merged
@IngmarVG-IB can you resolve the conflict if possible by then

**Comment by saiprasannasastry** [comment]
@JonasKs  maybe i can bump up for the other minor changes, i am just waiting to merge this and make a 0.5.0

**Comment by JonasKs** [comment]
That would be great, @saiprasannasatry🍺

**Comment by saiprasannasastry** [comment]
@JonasKs you want me to make a new version for the smaller checkins right? just confirming

**Comment by JonasKs** [comment]
Yeah, I’d love that. The `oslo.log` dependencies has started to grind my gears, and the fix has been in for a good while now😁

**Comment by IngmarVG-IB** [comment]
Latest changes should have addressed these issues

**Comment by IngmarVG-IB** [comment]
rebased and added txt records as well

**Comment by saiprasannasastry** [comment]
@IngmarVG-IB it still says merge conflict

**Comment by tc0nn** [comment]
Any chance we can get this going again?

**Comment by IngmarVG-IB** [comment]
We're working on a new branch that has all these changes + some included, expect to be complete in a few weeks. Check out https://github.com/rollant/infoblox-client/

**Pull request closed without merging**